src/NHibernate.Test/NHSpecificTest/NH901/Fixture.cs

```diff
@@ -81,7 +81,11 @@ public void ReplaceValueTypeComponentWithSameValueDoesNotMakeItDirty()
 				a.Street = "Some Street";
 				a.City = "Some City";
 				jimmy.Address = a;
-				Assert.AreNotSame(jimmy.Address, a);
+#pragma warning disable NUnit2040
+				// NUnit2040 : The SameAs constraint always fails on value types as the actual and the expected value cannot be the same reference
+				// TODO: Find out what was the intention of the test. Address is a struct.
+				Assert.That(a, Is.Not.SameAs(jimmy.Address));
+#pragma warning restore NUnit2040
```

Comment:
I think this can be removed. It would make more sense to check the structs are equal before assigning the new a to Jimmy's address.

Reply:
I wanted to do minimal changes. I think I rather disable this warning in editorconfig file.